paasta_tools/cli/cmds/security_check.py

```diff
@@ -27,8 +27,9 @@
 def add_subparser(subparsers):
     list_parser = subparsers.add_parser(
         'security-check',
-        description='Performs a security check (alpha)',
-        help='Performs a security check (alpha)',
+        description='Performs a security check consisting in a few tests like shellshock vulnerability, the user '
```

Comment:
Technically this is a bit yelp specific because the open source version could be doing anything. That said I don't think it's likely anyone outside yelp is using this script so I think it's fine to add this help here.

Also "consisting of" ;-)